File: PHP Excel Examples/CellsApiDemo/Column/insertColumns.php
<?php

use Spire\Cloud\Excel\Sdk\Api\CellsApi;
use Spire\Cloud\Excel\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey, $baseUrl);
$apiInstance = new CellsApi($configuration);

$name = "InsertColumns.xlsx";
$folder = "input";
$storage = null;
$sheetName = "Sheet1";
$columnIndex = 2;
$columns = 2;
$formatAs = "FormatAsBefore";
$apiInstance->insertColumns($name, $sheetName, $columnIndex, $columns, $formatAs, $folder, $storage);

File: Python Excel Examples/CellsApiDemo/column/insertColumns.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration
from spirecloudexcel.api.cells_api import CellsApi

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = ExcelConfiguration(appId, appKey,baseUrl)
api = spirecloudexcel.api.cells_api.CellsApi(configuration)
name = "InsertColumns_1.xlsx"
sheetName = "Sheet1"
column_index = 2  #index starts 1
columns = 2
format_as = "FormatAsBefore"  # FormatAsBefore FormatAsAfter FormatDefault.(default to FormatDefault)
storage = ""
folder = "/Cells/Column/"
api.insert_columns(name, sheet_name=sheetName, column_index=column_index,
                        columns=columns,format_as=format_as,
                        folder=folder,
                        storage=storage)